packages/terra-form-fieldset/src/Fieldset.jsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable react/jsx-boolean-value, jsx-a11y/label-has-for */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+
+import 'terra-base/lib/baseStyles';
+import styles from './Fieldset.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Children the Field contains.
+   */
+  children: PropTypes.node,
+  /**
+   * Error message for when the input is invalid.
+   */
+  error: PropTypes.node,
+  /**
+   * Help element to display with the input.
+   */
+  help: PropTypes.node,
```

Comment:
Should this also be `string`? Do we have a use case of someone needing to pass in something other than text?

Reply:
We are moving [terra-form/src/Fieldset.jsx](https://github.com/cerner/terra-core/blob/master/packages/terra-form/src/Fieldset.jsx) to its own package in this PR and that component had the help prop take a node. For passivity sake, I think we should keep this Fieldset component's help prop take a node as well.